docusaurus/docs/operate/morse_migration/11_testing.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migration Testing
+sidebar_position: 11
+---
+
+## Table Of Contents
+
+- [Background](#background)
+- [1. Retrieve a Morse TestNet Snapshot](#1-retrieve-a-morse-testnet-snapshot)
+- [2. Export Morse TestNet Export State](#2-export-morse-testnet-export-state)
+- [3. Transform & Merge Morse Exports to a Canonical Account State Import Message](#3-transform--merge-morse-exports-to-a-canonical-account-state-import-message)
+
+## Background
+
+For testing the migration process end-to-end, _on a live and public network and with real stakeholders_, the Morse state
+is initially imported into Shannon TestNet(s).
+
+In order to maximize both developer convenience and peace of mind, the Morse states which are imported into Shannon
+TestNet(s) are **a merge of both a Morse MainNet and a Morse TestNet state exports**.
+
+## 1. Retrieve a Morse TestNet Snapshot
+
+Morse state exports are derived from snapshots. Since Morse TestNet snapshots are not automated, like Morse MainNet
+snapshots are, Morse TestNet snapshots are taken manually and distributed via STORJ network.
```

Comment:
```suggestion
Morse state exports are derived from snapshots. Since TestNet snapshots are not automated, like MainNet, TestNet snapshots are taken manually and distributed via STORJ network.
```

Feels like too much occurrences of Morse XNet snapshots.